act-rules-format.bs

```diff
@@ -296,6 +296,19 @@ Even concepts like headings and images can be misunderstood. These terms could r
   </blockquote>
 </div>
 
+<div class=example>
+  <p>**Example:** The applicability of a rule with the page as a test target</a></p>
+  <blockquote>
+    <p>The rule applies to any page where the root element is an `html` element, and which is not embedded in another page.</p>
```

Comment:
I find it unclear whether the "which" refers to "any page" or "the root element". I think it needs a bit of clarification.